src/Sulu/Bundle/Sales/CoreBundle/Manager/CustomerTypeManager.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Sulu\Bundle\Sales\CoreBundle\Manager;
+
+use Sulu\Bundle\Sales\CoreBundle\Model\CustomerType;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class CustomerTypeManager
```

Comment:
Since this is implemented as a service you could inject translator the translator in a constructor.
then you wouldn't have to pass the translator to every function